docs/csharp/language-reference/keywords/readonly.md

```diff
@@ -14,7 +14,10 @@ ms.assetid: 2f8081f6-0de2-4903-898d-99696c48d2f4
 
 The `readonly` keyword is a modifier that can be used in three contexts:
 
-- In a [field declaration](#readonly-field-example), `readonly` indicates that assignment to the field can only occur as part of the declaration or in a constructor in the same class.
+- In a [field declaration](#readonly-field-example), `readonly` indicates that assignment to the field can only occur as part of the declaration or in a constructor in the same class. A readonly field can be assigned and reassigned multiple times within the field declaration and constructor. 
+  - If the `readonly` field is a value type, it is made immutable. 
```

Comment:
I thought about this change over the weekend. What do you think of this wording:

```suggestion
A `readonly` field cannot be assigned after the constructor exits. That has different implications for value types and reference types:
- Because value types directly contain their data, a field that is a  `readonly` value type is immutable. 
```